manifests/init.pp

```diff
@@ -94,6 +98,7 @@
 class pulpcore (
   Stdlib::Absolutepath $cache_dir = '/var/lib/pulp/tmp',
   Stdlib::Absolutepath $config_dir = '/etc/pulp',
+  Stdlib::Absolutepath $libexecdir = '/usr/libexec/pulpcore',
```

Comment:
If these are not part of the package, the FHS states that they should be under `/usr/local/libexec/pulpcore` instead.

Reply:
You could make the same argument about `/var/lib/pulp`. I also would thing that managing the pulp user as part of the RPM and the service files would be good, but I think the ansible-based installer cares about pip installs so it probably won't happen.